h5p-bildetema/src/components/LanguageSelector/LanguageSelector.module.scss

```diff
@@ -0,0 +1,9 @@
+.languageSelector {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.language_select {
+  width: 50%;
+  margin-bottom: 0.8rem;
```

Comment:
Use `gap` on the parent element to decide how much space should be between the language elements.